load_uploaded_files.json

```diff
@@ -0,0 +1,65 @@
+{
+    "id": "load_uploaded_files",
+    "summary": "Load files from the user workspace.",
+    "description": "Loads user-uploaded files from the local user workspace / data store. The files must have been stored by the authenticated user on the back-end currently connected to.",
+    "categories": [
+        "cubes",
+        "import"
+	],
+    "parameter_order": [
+        "paths",
+        "format",
+        "options"
+    ],
+    "parameters": {
+        "paths": {
+            "description": "The file(s) and folder(s) to read.",
+            "schema": {
+				"type": "array",
+				"subtype": "file-folder-paths",
+				"items": {
+					"anyOf": [
+						{
+							"type": "string",
+							"subtype": "file-path"
+						},
+						{
+							"type": "string",
+							"subtype": "folder-path"
+						}
+					]
+				}
+			},
+            "required": true
+        },
+        "format": {
+            "description": "The file format to save to. It must be one of the values that the server reports as supported input file formats, which usually correspond to the short GDAL/OGR codes. This parameter is *case insensitive*.",
```

Comment:
Shouldn't this be 'file format to read from', or am I missing something?

Reply:
Oh, yes, thanks. I coped from save_result and forgot to change that part. Fixed.